Sources/FigmaExportCore/Helpers/Zip4Sequence.swift

```diff
@@ -0,0 +1,203 @@
+
+/// Creates a sequence of tuples built out of 4 underlying sequences.
+///
+/// In the `Zip4Sequence` instance returned by this function, the elements of
+/// the *i*th tuple are the *i*th elements of each underlying sequence. The
+/// following example uses the `zip(_:_:)` function to iterate over an array
+/// of strings and a countable range at the same time:
```

Comment:
Where did you copy the code from in this file?

Reply:
I indicated this in the commit comment. Duplicate here:
https://github.com/amomchilov/ZipNsequence/blob/master/Zip4Sequence.swift